wand/image.py

```diff
@@ -1028,6 +1110,51 @@ def background_color(self, color):
             if not result:
                 self.raise_exception()
 
+    def matte(self, flag):
```

Comment:
This is a tricky method as it feels like it should be a property. It maps to the CLI `-matte` & `+matte` operations which makes us want to use a more descriptive name. Perhaps I can updated the document to help describe the effects of enabling / disabling a matte channel?